src/env/mainnet.ts

```diff
@@ -0,0 +1,5 @@
+export const port = +process.env.PORT || 8080;
+export const jwtSecret = process.env.JWT_SECRET || "your-secret-here";
```

Comment:
For production I would suggest throwing if the secret isn't set instead of providing a default value. This might encourage people to use an insecure secret by accident.